cylc/flow/time_parser.py

```diff
@@ -374,7 +375,7 @@ def _get_point_from_expression(self, expr, context, is_required=False,
             expr_to_parse = expr + "00"
         try:
             expr_point = self.timepoint_parser.parse(expr_to_parse)
-        except ValueError:
+        except ValueError:  # not IsodatetimeError as too specific
```

Comment:
A couple of unit tests rely on this being a `ValueError`, as they end up asking the timepoint parser to parse an `expr_to_parse` that is not a timepoint. Thus a genuine `ValueError` occurs in isodatetime - not an artificially raised exception